osbs/build/build_requestv2.py

```diff
@@ -331,6 +344,8 @@ def set_data_from_reactor_config(self, reactor_config_data):
         self.set_pull_registries(reactor_config_data)
         self.set_registry_organization(reactor_config_data)
         self.set_required_secrets(reactor_config_data)
+        if reactor_config_data:
```

Comment:
This `if` seems suspicious, all methods here works with `reactor_config_data`

Reply:
removed